packages/core/components/InputOrGroup/fields/ArrayField/index.tsx

```diff
@@ -196,6 +196,7 @@ export const ArrayField = ({
                               }
                             }}
                             className={getClassNameItem("summary")}
+                            tabIndex={0}
```

Comment:
Hmm, in some scenarios ArrayFields seem to be tabbable twice, but I can't consistently reproduce. 

Each focus state seems to have its own focus style 🤔 

Perhaps using a `button` for the item would work? Any reason you didn't already try that?

Reply:
Yeah this is a temp "fix", as noted in the description above, needs more work

> keyboard a11y & a11y of nested interactive controls in ArrayField

 Using `button` would definitely be better, but the bigger issue is the action button(s) (e.g. delete) cannot be nested inside.

Reply:
I've removed the `tabIndex={0}` here, as it's not solving the a11y problem anyway – I mainly put it on there so I can get a focus state to style – and will spin out an issue to address properly.